src/Hint/NewTypeDerivingStrategies.hs

```diff
@@ -0,0 +1,38 @@
+{-
+    Suggest using explicit deriving strategies for newtype instances
+
+<TEST>
+newtype Foo = Foo Int deriving (Show, Eq) -- newtype Foo = Foo Int deriving newtype (Show, Eq)
+newtype Foo = Foo { getFoo :: Int } deriving (Show, Eq) -- newtype Foo = Foo { getFoo :: Int } deriving newtype (Show, Eq)
+newtype Foo = Foo Int deriving stock Show
+</TEST>
+-}
+module Hint.NewTypeDerivingStrategies (newTypeDerivingStrategiesHint) where
+
+import Hint.Type
+
+newTypeDerivingStrategiesHint :: DeclHint
+newTypeDerivingStrategiesHint _ _ = newTypeDerivingStrategiesHintDecl
+
+newTypeDerivingStrategiesHintDecl :: Decl_ -> [Idea]
+newTypeDerivingStrategiesHintDecl x
+    | Just new <- newtypeDecl x
+    = [suggestN "Use DerivingStrategies" x new
```

Comment:
Instead of `suggestN` you need to define `ignoreN`.

Reply:
:ok_hand: